packages/geolocation/__tests__/reverseGeocoding.test.js

```diff
@@ -0,0 +1,41 @@
+const boom = require('@hapi/boom');
+const { reverseGeocoding } = require('../src/index');
+
+test('should return a place, given a longitude and latitude', async () => {

```

Comment:
@DilanAriza We need to bear on mind that tests should be written using BDD with the given-when-then approach